src/Sylius/Behat/Context/Api/BaseContext.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Behat\Context\Api;
+
+use Behat\Behat\Context\Context;
+use Behat\Behat\Hook\Scope\BeforeStepScope;
+use Sylius\Behat\Service\SharedStorageInterface;
+
+abstract class BaseContext implements Context
```

Comment:
Why is this an abstract? You could just add it as a normal context to the suites, maybe even check if it can be added globally in the API context. Or you could just convert it into a trait.